CONTRIBUTING.md

```diff
@@ -51,6 +51,12 @@ cannot be merged.
 
 ([source](http://blog.qt.io/blog/2014/06/13/qt-weekly-13-qstringliteral/))
 
+## Use .arg(arg1, arg2) instead of .arg(arg1).arg(arg2)
+
+If you just need to use a plain .arg() call, use multiple arguments within it and not as separate .arg() calls as that'll avoid creating temporary QString objects.
+
```

Comment:
With the proviso that if there are two or three arguments and they are all numeric then force the second and third to be expressed as `QStrings` first {with `QString::number(`...`)`} least they be interpreted as _width_ and _base_ specifiers for the first number if seen as numbers! 🤓